.github/workflows/dev.yml

```diff
@@ -285,14 +285,6 @@ jobs:
             target=x86_64-unknown-linux-gnu \
             cargo doc
 
-      - id: "build_each_commit"
-        continue-on-error: true
-        name: "build each commit"
-        if: "${{ github.event.pull_request || github.event.merge_group }}"
-        run: |
-          BASE="${{ github.event.pull_request.base.sha || github.event.merge_group.base.sha }}"
-          just build-sweep "${BASE}"
-
```

Comment:
OK to remove, but I don't fully understand the explanations from commit description. How is this broken, and who is it not actionable?